front/lib/data_sources.ts

```diff
@@ -57,8 +57,17 @@ export function isWebsite(
   return ds.connectorProvider === "webcrawler";
 }
 
+export function isManagedConnectorProvider(
+  connectorProvider: ConnectorProvider
```

Comment:
Can we use the function right above?

Reply:
I would if I had a datasource but in the caller you can notice that I only have a string.